docs/larq/guides/bnn-architecture.md

```diff
@@ -94,20 +90,18 @@ def conv_with_shortcut(x):
 
 ## Pooling
 
-The [XNOR-net authors](https://arxiv.org/abs/1603.05279) found that accuracy improves when applying batch normalization after instead of before max-pooling. In general, max pooling in BNNs can be problematic as it can lead to skewed binarized activations.
+The *XNOR-net* ( [paper](https://arxiv.org/abs/1603.05279) | [larq zoo model](/zoo/api/literature/#xnornet) ) authors found that accuracy improves when applying batch normalization after instead of before max-pooling. In general, max pooling in BNNs can be problematic as it can lead to skewed binarized activations.
```

Comment:
```suggestion
The *XNOR-net* ([paper](https://arxiv.org/abs/1603.05279); [Larq Zoo model](/zoo/api/literature/#xnornet)) authors found that accuracy improves when applying batch normalization after instead of before max-pooling. In general, max pooling in BNNs can be problematic as it can lead to skewed binarized activations.
```